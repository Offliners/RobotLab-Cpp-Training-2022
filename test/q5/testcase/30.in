(7/16/6/5)=*19*(15*7/16)=+10/14+16-19/6/18+1+(14/2)/2+3/9-7-15=+8*13=+12-17*7-18%3=/((4-13)+14+19*19)*8/3%4=*((5*12+9)*1-16)/6+20%12=-((20+14+12*5-18+7)*(4*12)-10-11+15*14/10/4+1+(3-8)*12)/(12+3+17)+14*16-19*(17/17+20+6*9+14-11-18-5/3/14/9*12/1+15/1-(1*10))=
