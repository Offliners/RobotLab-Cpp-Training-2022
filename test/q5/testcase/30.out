1379
3
36
-462
